plugins/hue/README.md

```diff
@@ -1,103 +1,372 @@
 # Phillips HUE
 
-# Requirements
-
-Needs httplib
+###New development of Hue plugin for use in smarthome.py (C) Michael Würtenberger 2014, 2015
+version 1.5 develop
 
 ## Supported Hardware
-
-Philips hue bridge
+Philips hue bridge, multiple bridges allowed
 
 # Configuration
-
 ## plugin.conf
-
-Typical configuration
+Typical configuration for 3 bridges
+<pre>
+[HUE]
+    class_name = HUE
+    class_path = plugins.hue
+    hue_user = 38f625a739562a8bd261ab9c7f5e62c8, 38f625a739562a8bd261ab9c7f5e62c8, 38f625a739562a8bd261ab9c7f5e62c8
+    hue_ip = 192.168.2.2,192.168.2.3,192.168.2.4
+    hue_port = 80,80,80
+    cycle_lamps = 3
```

Comment:
I suggest to use the standard separator, which is a pipe, when you want to cinfigure lists, which will also provide the values as list. No need to manualy split the string.